docs/integrations/platforms/kubernetes.mdx

```diff
@@ -105,11 +127,53 @@ Default re-sync interval is every 1 minute.
 </Accordion>
 
 <Accordion title="authentication">
-  This block defines the method that will be used to authenticate with Infisical so that secrets can be fetched. Currently, only [Service Tokens](../../documentation/platform/token) can be used to authenticate with Infisical. 
+  This block defines the method that will be used to authenticate with Infisical so that secrets can be fetched
+</Accordion>
+
+<Accordion title="authentication.universalAuthMachineIdentity">
+  The universal machine identity authentication method is used to authenticate with Infisical. The client ID and client secret needs to be stored in a Kubernetes secret. This block defines the reference to the name and namespace of secret that stores these credentials.
+
+  #### 1. Create a machine identity
+  You need to create a machine identity, and give it access to the project(s) you want to interact with. You can [read more about machine identities here](/documentation/platform/identities/universal-auth).
+
+  #### 2. Create Kubernetes secret containing machine identity credentials
+
+  Once you have created your machine identity and added it to your project(s), you will need to create a Kubernetes secret containing the identity credentials.
+  To quickly create a Kubernetes secret containing the identity credentials, you can run the command below. Make sure you replace `<your-identity-client-id>` with the identity client ID and `<your-identity-client-secret>` with the identity client secret.
+
+  ``` bash
+    kubectl create secret generic universal-auth-credentials --from-literal=clientId="<your-identity-client-id>" --from-literal=clientSecret="<your-identity-client-secret>"
+  ```
+
+  #### 3. Add reference for the Kubernetes secret containing the identity credentials
+  Once the secret is created, add the name and namespace of the secret that was just created under `authentication.serviceToken.universalAuthMachineIdentity` field in the InfisicalSecret resource.
```

Comment:
Did you mean 

universalAuthMachineIdentity.credentials.secretName 

universalAuthMachineIdentity.credentials.secretNamespace